### ISSUE TITLE
"Fallout 4 has stopped working" seconds after clicking launch, Vanilla, MO2

### ISSUE THREAD

**Issue by Kanzuke117** [TanninOne/modorganizer#416]
I just grabbed the [latest release](https://github.com/TanninOne/modorganizer/releases/tag/v2.0.2a) after having issues with NMM's use of Symlinks and sounds. Like Skyrim, I've installed it in the same directory as Fallout 4, except not on C:\ as I don't need the SSD's speed boost to brute force virtualizing ~300 mods yet.

All I've done so far in terms of configuring MO past a clean install is moving all the mod archives from NMM's directories to MO's, querying mod info and changing the default skin. 

I've seen many an issue here already regarding crashes on the Vanilla game, but every one so far seems to describe at least making it to a black screen or main menu. [I don't even make it that far](http://puu.sh/lKaxv/024e4d9ef4.mp4).

[usvfs-2015-12-05_13-11-57.txt](https://github.com/TanninOne/modorganizer/files/52847/usvfs-2015-12-05_13-11-57.txt) and [mo_interface.txt](https://github.com/TanninOne/modorganizer/files/52848/mo_interface.txt) generated from opening .exe, clicking launch, clicking close the program when it stops working and then exiting Mod Organizer. [Upload of the crash dump mentioned in the usvfs.log](http://puu.sh/lKaSt/d52a910369.dmp), which gives Exception Code 0xC0000005, "The thread tried to read from or write to a virtual address for which it does not have the appropriate access". No heap or error information. For reference, I am running everything from an Administrator account, and specifically running the executable as administrator gave no differing results.

I saw a few issues where installing Mod Organizer in C:\ or C:\Program Files was recommended, and so tried these directories, with a completely vanilla Mod Organizer trying to launch a completely vanilla Skyrim, and saw the same results both times as seen in the video. I could try moving the game to C:/ as well if there are no other suggestions, I'd just prefer not to as I'd have to move quite a few things off to make the space.

**Comment by Grogwort** [comment]
Same here. Windows 7 x64, MO a2, Fallout 1.2.37, Mods or No Mods
Launching Fallout4 or Fallout4Launcher from within MO.

Problem Event Name:	APPCRASH
Application Name:	Fallout4.exe
Application Version:	1.2.37.0
Application Timestamp:	565385f8
Fault Module Name:	usvfs_x64.dll
Fault Module Version:	0.1.0.0
Fault Module Timestamp:	565b23c4
Exception Code:	c0000005
Exception Offset:	00000000000f152d

**Comment by unobtainer** [comment]
The current MO2 alpha seems to produce this crash on Win 7x64.

**Comment by michaelrw10** [comment]
Winodws 10 Pro (x64) and I am having very few problems, but overall it is functioning very well. I have been using MO for the past 3 years and am very experienced with it, so maybe that helps. Not sure why others seem to behaving all this trouble. Nonetheless, I will report the two problems I do have.

1. When launching MO, an error is thrown
"failed to initialize plugin <path>/Mod Organizer/plugins/pyCfg.py: An unexpected C++ exception was thrown in python code"

* this is easy to fix by disabling the plugin; or just leave it enabled and live with the error. It does not prevent MO from working. (fwiw, i have python installed 2.7 installed on my machine)

2. local savegames doesn't work (but you already know this)

Aside from these two things, my experience with MO2 is identical to that of MO for that I use for skyrim/oblivion/fallout3. Great job tannin, keep up the hard work, it is much appreciated by the community, I hope the python error makes sense to you

**Comment by grasmann** [comment]
@michaelrw10 
All your problems are known issues.

1. There's a missing file that results in the python stuff not initializing correctly.
Error can be ignored.

2. In the alpha 2 the overwrite folder isn't used.
Also in alpha 2 the 32bit support is broken.
I guess that's why FO4Edit isn't working.

3. Yeah local savegames isn't working.
To be honest I don't even know what that is supposed to be.

The reason why you're not experiencing any problems, just like me, is because you're using windows 8/10.

**Comment by michaelrw10** [comment]
@grasmann - i was merely explaining my experience, i was not making a formal report of a bug, which is why i didnt create a new report in the tracker. just figured i would share my own experiences, even though people can read what the 'known' bugs are -- not everyone experiences the same thing.

At any rate, there was just an update to Steam (not sure if that is related to this or not) but now I am experiencing what is being described here. The game crashed shortly after launching, stating that the game has stopped working and windows is trying to find a solution. It does not matter if I launch the fallout exe, or if i use F4SE and launch that way. 

However, I switched the launch mechanism to 'script extender' option, and now it seems to work again. I also confirmed that the steam app id of the game didnt change. 

Strangest thing is, i reverted the setting back to Mod Organizer for the launch mech, and no error, no crash when I open. Hmmm.. very odd behavior. 

I wish there was something I could do to help tannin because i simply love MO and use it for all of my modded games. it is an engineered work of software art, it would be a shame if we ever lost it. Im sure tannin has a real job and that MO is surely not his main priority in life. I would totally buy a license for MO to help support it though. but i digress

**Comment by michaelrw10** [comment]
well, now it is not working again. I have literally done nothing aside from exit and restart the game. have made no mod changes, not file changes to the game directory, nothing. There was a windows update, along with the steam update that i already mentioned, but im not sure if thats related. I unstalled the windows update to revert back, but it did not help. I also tried to toggle the launch mechanism, but it did not work for me this time. I am at a loss. Looks like i will have to use NMM.. the thought of which is disgusting. irritating as all hell... i do not like having unsolvable problems.

**Comment by sprocketaudio** [comment]
I had this too after moving from NMM. Fixed it my backing up my ini files, and letting steam do an integretiy check on the game... found some missing files.. now thing seem to be working! :)

**Comment by michaelrw10** [comment]
yeah ive done all of that. multiple times.to no avail. I am currently in a state where I can launch MO and it will work (if it is in a good mood). Other times it is in a bad mood and will cause crash after 2 seconds
